app/javascript/packs/MainApp/components/RefereeProfile.jsx

```diff
@@ -7,50 +7,151 @@ class RefereeProfile extends Component {
     match: PropTypes.shape({
       params: PropTypes.object
     }).isRequired
-  }
+  };
 
   state = {
+    httpStatus: 200,
+    httpStatusText: '',
     firstName: '',
     lastName: '',
     bio: '',
-    email: ''
-  }
+    email: '',
+    pronouns: '',
+    nationalGoverningBodies: [],
+    certifications: []
+  };
 
   componentDidMount() {
-    const { match: { params } } = this.props
+    const { match: { params } } = this.props;
 
     axios.get(`/api/v1/referees/${params.id}`)
-      .then(({ data }) => {
-        const { attributes } = data.data
+      .then(({ status, statusText, data }) => {
+        const { data: { attributes }, included } = data;
+        const certifications = included
+          .filter(({ type }) => type === 'certification')
+          .map(certification => certification.attributes);
+        const nationalGoverningBodies = included
+          .filter(({ type }) => type === 'national_governing_body')
+          .map(nationalGoverningBody => nationalGoverningBody.attributes);
         this.setState({
+          httpStatus: status,
+          httpStatusText: statusText,
           firstName: attributes.first_name,
           lastName: attributes.last_name,
           bio: attributes.bio,
-          email: attributes.email
+          email: attributes.email,
+          pronouns: attributes.pronouns,
+          nationalGoverningBodies,
+          certifications
         })
       })
+      .catch(({ response: { status, statusText } }) => {
```

Comment:
similar helper function thought here, in theory we could decide to change the error message that is shown for the user so would be good to keep that logic insulated in it's own function.